polldaddy-shortcode.php

```diff
@@ -448,7 +448,7 @@ function polldaddy_link( $content ) {
 		$textarr = wp_html_split( $content );
 		unset( $content );
 		foreach( $textarr as &$element ) {
-			if ( '' === $element || '<' === $element{0} )
+			if ( '' === $element || '<' === $element[0] )
```

Comment:
LGTM! I assume at this point we're already certain that there is some `$content` so the array won't be empty (or at least, never happened up until now) 🚢

Reply:
I haven't looked in to that. Just wanted to silence the warning.